sentinel-transport/sentinel-transport-common/src/main/java/com/alibaba/csp/sentinel/command/annotation/CommandMapping.java

```diff
@@ -30,4 +26,7 @@
 public @interface CommandMapping {
 
     String name();
+
+    String desc();
```

Comment:
Maybe it's better to be optional (default empty)?

Reply:
Define the desc to be not empty aims to force user create CommandHandler with a detailed description. If set default empty when user get the api list they don't know what the exactly usage of the api. So I would prefer it not be default empty.

Reply:
Okay, it's reasonable.